### PR TITLE
[chore|sde-backend] Docker notice.md added

### PR DESCRIPTION
**Pull request by adityagajbhiye9**
## Description

- Docker notice.md added.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by adkumar1** [comment]
Need to remove "_Notice for Docker image_" from README.md
& create a link in README.md referering DOCKER_NOTICE.md

**Comment by adkumar1** [comment]
Looks fine now

**Review by adkumar1 — APPROVED** [review]

**Review by hzierer — CHANGES_REQUESTED** [review]
- I think you have to adjust your workflow too to include the newly created file instead of the default README.
see the last line ( readme-filepath ... ) [here](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-06#github-workflow-step)
- please rename the file to `notice.md` as stated in the TRG

**Comment by adityagajbhiye9** [comment]
@hzierer  As notice file already exit created the docker_notice.md file.

**Comment by hzierer** [comment]
> @hzierer As notice file already exit created the docker_notice.md file.

yeah, sorry my bad. Haven't described it in detail enough. The "notice for docker" has to be in the same directory as the Docker file - so it is no problem to be named notice.md.

Sorry for any inconveniences caused.

**Comment by adityagajbhiye9** [comment]
@hzierer please review.

**Comment by adityagajbhiye9** [comment]
> > @hzierer As notice file already exit created the docker_notice.md file.
> 
> yeah, sorry my bad. Haven't described it in detail enough. The "notice for docker" has to be in the same directory as the Docker file - so it is no problem to be named notice.md.
> 
> Sorry for any inconveniences caused.

No worries :)

File: .github/workflows/dockerbuild.yaml [hunk 1]
```diff
@@ -99,4 +99,6 @@ jobs:
            # readme-filepath: path/to/dedicated/notice-for-docker-image.md 
            username: ${{ secrets.DOCKER_HUB_USER }}
            password: ${{ secrets.DOCKER_HUB_TOKEN }}
-           repository: ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}
\ No newline at end of file
+           repository: ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}
+           # IMPORTANT: Adjust this to the actual path of your container image notice
+           readme-filepath: ./docker_notice.md
```
[the remark this answers is not in the document]
Comment (adityagajbhiye9) [reply .github/workflows/dockerbuild.yaml:104]: renamed it.

**Review by hzierer — APPROVED** [review]

**Review by FaGru3n — APPROVED** [review]
LGTM

**Comment by FaGru3n** [comment]
sorry missed a upper case link in this PR that is not nessesary... so created a new one to fix but also don´t know your workflow how to change theese files... 

will close it... right now https://github.com/eclipse-tractusx/managed-simple-data-exchanger-backend/pull/105

**Pull request merged**